docs/Components/Sensors/LiDARSensor/LiDARSensor/index.md

```diff
@@ -143,29 +143,44 @@ Example of how to get XYZ point cloud data:
+| Pcl 48 | 48-byte extended version point cloud format used by *Autoware* | XYZ_VEC3_F32, PADDING_32, INTENSITY_F32, RING_ID_U16, PADDING_16, AZIMUTH_F32, DISTANCE_F32, RETURN_TYPE_U8, PADDING_8, PADDING_16, PADDING_32, TIME_STAMP_F64 |
+| ML Instance Segmentation | Machine learning format for instance/semantic segmentation tasks | XYZ_VEC3_F32, ENTITY_ID_I32, INTENSITY_F32 |
+| Radar Smart Micro | Format used in Radar Smart Micro | XYZ_VEC3_F32, RADIAL_SPEED_F32, POWER_F32, RCS_F32, NOISE_F32, SNR_F32 |
+| Custom | Empty format that allows the user to define its fieldsets |  |
 
 !!! note "*rosPCL48* format"
     For a better understanding of the *rosPCL48* format, we encourage you to familiarize yourself with the point cloud pre-processing process in *Autoware*, which is described [here](https://autowarefoundation.github.io/autoware-documentation/latest/design/autoware-architecture/sensing/data-types/point-cloud/#channel).
 
 #### Elements configurable from the editor level
-- `Pcl 24 Topic` - the *ROS2* topic on which the [`PointCloud2`][pointcloud2] message is published<br>(default: `"lidar/pointcloud"`)
-- `Pcl 48 Topic` - the *ROS2* topic on which the [`PointCloud2`][pointcloud2] message is published<br>(default: `"lidar/pointcloud_ex"`)
 - `Frame ID` - frame in which data are published, used in [`Header`](https://docs.ros2.org/latest/api/std_msgs/msg/Header.html) (default: `"world"`)
-- `Publish PCL24` - if publish cloud *PCL24* (default: `true`)
-- `Publish PCL48` - if publish cloud *PCL48* (default: `true`)
-- `Qos Settings` - Quality of service profile used in the publication<br>(default: `Best effort`, `Volatile`, `Keep last`, `5`)
-
-#### Published Topics
+- `Qos` - Quality of service profile used in the publication
+    - `Reliability Policy` - Reliability policy (default: `Best effort`)
+    - `Durability Policy` - Durability policy (default: `Volatile`)
+    - `History Policy` - History policy (default: `Keep last`)
+    - `History Depth` - History depth. If history policy is `Keep all`, depth is ignored. (default: `5`)
+- `Point Cloud 2 Publishers` - List of [sensor_msgs/PointCloud2 message](https://docs.ros2.org/latest/api/sensor_msgs/msg/PointCloud2.html) publishers
+    - `Topic` - Topic name to publish on
+    - `Publish` - If false, publishing will be stopped
+    - `Fields Preset` - allows selecting one of the pre-defined fieldsets (choose `Custom` to define your own)
+    - `Fields` - List of fields to be present in the message
+- `Radar Scan Publishers` - List of [radar_msgs/RadarScan message](http://docs.ros.org/en/noetic/api/radar_msgs/html/msg/RadarScan.html) publishers
+    - `Topic` - Topic name to publish on
+    - `Publish` - If false, publishing will be stopped
+
+!!! note "Elements configurable in simulation runtime"
+    Once the simulation starts, only the `Publish` flag is handled. All of the publishers are initialized on the simulation startup and updates of their parameters are not supported in runtime.
```

Comment:
I would maybe add that "are not supported" means "will not modify publishing configuration" and not "will cause some random crash".